api/app/__init__.py

```diff
@@ -22,22 +20,15 @@
 
 
 def create_app():
-    is_production = os.environ.get("ENVIRONMENT") == "PRODUCTION"
-    if is_production:
-        config = ProductionConfig()
-    else:
-        config = DevelopmentConfig()
-        # initialize dev database here?
+    environment = os.environ.get("ENVIRONMENT")
```

Comment:
@bnorquist, this is ready to be merged in but I think once you stashed the changes you also removed changing this from `ENVIRONMENT` to `FLASK_ENV`